www/src/views/starter/meta.js

```diff
@@ -0,0 +1,174 @@
+        </div>
+      </div>
+
+      <div
+        css={{
+          marginTop: rhythm(3 / 4),
+          marginRight: 15,
+          display: "flex",
+          flexWrap: "nowrap",
+          flexGrow: 1,
+          justifyContent: "space-between",
+          borderBottom: `1px solid ${colors.ui.light}`,
+          paddingBottom: rhythm(2 / 4),
+          [presets.Phablet]: {
+            borderBottom: `0px`,
+          },
+        }}
+      >
+        <div
+          css={{
+            paddingRight: 15,
+            paddingBottom: 15,
+            whiteSpace: "nowrap",
+            overflow: "hidden",
+            textOverflow: "ellipsis",
+          }}
+        >
+          <span css={{ color: colors.gray.light }}>{`By  `}</span>
+          <a
+            css={{
+              "&&": {
+                boxShadow: `none`,
+                borderBottom: 0,
+                color: colors.lilac,
+                cursor: `pointer`,
+                fontFamily: options.headerFontFamily.join(`,`),
+                "&:hover": {
+                  background: `transparent`,
+                  color: colors.gatsby,
+                },
+              },
+            }}
+            href={`https://github.com/${starter.owner.login}`}
+          >
+            {starter.owner.login}
+          </a>
+        </div>
+        <div
+          css={{
+            flex: "2 0 auto",
+            textAlign: "right",
+            position: "relative",
+            zIndex: 1,
+          }}
+        >
+          <a
+            href={frontmatter.demo}
+            css={{
+              border: 0,
+              borderRadius: presets.radius,
+              color: colors.accent,
+              fontFamily: options.headerFontFamily.join(`,`),
+              fontWeight: `bold`,
+              marginRight: rhythm(1.5 / 4),
+              padding: `${rhythm(1 / 6)} ${rhythm(2 / 3)}`, // @todo same as site showcase but wrong for some reason
+              textDecoration: `none`,
+              WebkitFontSmoothing: `antialiased`,
+              "&&": {
+                backgroundColor: colors.accent,
+                borderBottom: `none`,
+                boxShadow: `none`,
+                color: colors.gatsby,
+                "&:hover": {
+                  backgroundColor: colors.accent,
+                },
+              },
+            }}
+          >
+            <MdLaunch
+              style={{
+                verticalAlign: `sub`,
+              }}
+            />
+            {` Visit demo `}
+          </a>
+          <ShareMenu
+            url={`https://github.com/${starter.githubFullName}`}
+            title={`Check out ${repoName} on the @Gatsby Starter Showcase!`}
+            image={imageSharp.childImageSharp.resize.src}
+            theme={"accent"}
+          />
+        </div>
+      </div>
+    </div>
+  )
+}
+
+function showDate(dt) {
+  const date = new Date(dt)
+  return date.toLocaleDateString(`en-US`, {
```

Comment:
Can we use the `graphql` date format function here and pass this in? Or not possible?

(not a big deal either way, just asking!)

Reply:
Possible! I haven't looked into why this was done this way, I just relocated what was already there for now. Good thought.